### PR TITLE
Training/task3 dom manipulation

### PR DESCRIPTION
**Pull request by Asim-Khaleel**
Completed 3rd task for editing the experience section

### CODE REVIEW

File: PageEdit/PageEdit.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Edit info</title>
+  <link rel="stylesheet" type="text/css" href="PageEdit.css" />
+</head>
+
+<body>
```
Comment (sikandar-ramzan) [inline PageEdit/PageEdit.html:10]: **Read about semantic html and refactor your markup (html) such that it reflects that semantic elements are used in the markup with proper hierarchical breakdown.**

File: PageEdit/PageEdit.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const linkToPortfolioPage = "../Portfolio/Portfolio.html";
```
Comment (sikandar-ramzan) [inline PageEdit/PageEdit.js]: use pascal case for constants

File: Portfolio/Portfolio.html [hunk 3]
```diff
@@ -9,7 +9,8 @@
   <body>
     <div class="informationContainer">
       <div class="loginSection">
-        <button class="loginText" id="loginButton" onclick="redirectToSignupPage()">Login</button>
+        <button class="loginText" onclick="redirectToEditPage()">Edit</button>
+        <button class="loginText" onclick="redirectToSignupPage()">Signup</button>
```
Comment (sikandar-ramzan) [inline Portfolio/Portfolio.html:13]: use consistent programming patterns/styles. In one file you are using javascript event handlers. Here you are using html inline events. Use one approach consistently throught your codebase.

File: Portfolio/Portfolio.js [hunk 4]
```diff
@@ -1,4 +1,10 @@
+const linkToSignupPage = "../Signup/Signup.html";
+const linkToEditPage = "../PageEdit/PageEdit.html";
```
Comment (sikandar-ramzan) [inline Portfolio/Portfolio.js]: Use PASCAL_CASE for your constants.

**Review by sikandar-ramzan — CHANGES_REQUESTED** [review]

File: PageEdit/PageEdit.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  saveExperienceButton.addEventListener("click", () => {
+    const companyName = document.getElementById("companyName").value;
+    const startDate = document.getElementById("startDate").value;
+    const endDate = document.getElementById("endDate").value;
+    const description = document.getElementById("description").value;
+
+    if (companyName === "" || startDate === "" || endDate === "" || description === "") {
+      alert("Please fill in all fields.");
+      return;
+  }
```
Comment (sikandar-ramzan) [inline PageEdit/PageEdit.js]: dont need this, you are setting all the input fields as required in the markup. This is redundant.

File: PageEdit/PageEdit.js (hunk 2 above)
Comment (Asim-Khaleel) [reply PageEdit/PageEdit.js]: Fixed, Thank you

File: Portfolio/Portfolio.html (hunk 3 above)
Comment (Asim-Khaleel) [reply Portfolio/Portfolio.html:13]: Fixed, Thank you

File: Portfolio/Portfolio.js (hunk 4 above)
Comment (Asim-Khaleel) [reply Portfolio/Portfolio.js]: Fixed. Thank you

File: PageEdit/PageEdit.js (hunk 5 above)
Comment (Asim-Khaleel) [reply PageEdit/PageEdit.js]: I removed the check for here. Thank you

File: Signup/Signup.js [hunk 6]
```diff
@@ -1,13 +1,12 @@
-const linkToPortfolioPage = '../Portfolio/Portfolio.html';
+const LinkToPortfolioPage = '../Portfolio/Portfolio.html';
```
Comment (sikandar-ramzan) [inline Signup/Signup.js]: use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case

File: PageEdit/PageEdit.js (hunk 2 above)
Comment (sikandar-ramzan) [reply PageEdit/PageEdit.js]: use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case

File: Portfolio/Portfolio.js (hunk 4 above)
Comment (sikandar-ramzan) [reply Portfolio/Portfolio.js]: use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case however i wrote the requirement "PASCAL_CASE" which is actually all caps.

File: Signup/Signup.js (hunk 6 above)
Comment (Asim-Khaleel) [reply Signup/Signup.js]: Sure I will. Please let me know when you are done commenting

File: PageEdit/PageEdit.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+  saveExperienceButton.addEventListener("click", () => {
+    const companyName = document.getElementById("companyName").value;
+    const startDate = document.getElementById("startDate").value;
+    const endDate = document.getElementById("endDate").value;
+    const description = document.getElementById("description").value;
+
+    const experienceEntry = document.createElement("div");
+    experienceEntry.classList.add("experience-entry");
+    experienceEntry.style.margin = "10px 0px";
```
Comment (sikandar-ramzan) [inline PageEdit/PageEdit.js]: why adding a css class to the element and then also setting css styles explicitly below that?

File: Signup/Signup.js (hunk 6 above)
Comment (sikandar-ramzan) [reply Signup/Signup.js]: @Asim-Khaleel review the new comments and fix all

Comment (Asim-Khaleel) [reply Signup/Signup.js]: @sikandar-ramzan I have resolve the comments. Please review them

**Review by sikandar-ramzan — APPROVED** [review]

**Pull request merged**